src/SignalR/clients/ts/signalr-protocol-msgpack/package.json

```diff
@@ -42,7 +42,7 @@
     "src/**/*"
   ],
   "dependencies": {
-    "@aspnet/signalr": "file:../signalr",
+    "@aspnet/signalr": "link:../signalr",
```

Comment:
Don't you need to update the msgpack npmproj now?

Reply:
Blazor project?

Reply:
https://github.com/aspnet/AspNetCore/blob/34a68d5f4860e1e5767839695e45f1eeda4c9f2d/src/SignalR/clients/ts/signalr-protocol-msgpack/signalr-protocol-msgpack.npmproj#L17